Dhruv: Added colab file [to implement scam classification]

Dhruv-2020EE30592 opened the pull request:
Colab file implementing deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B

Is able to distinguish between important and not important mails in some cases (extensive testing has not been done)

Not able to correctly identify scam mails [prompt engineering]

Dhruv-2020EE30592 committed:
Dhruv: Added colab file [to implement scam classification]

SwordAndTea reviewed (commented):
add a spereate folder to store those notebook files instead of put it directly in the root dir.

cliaker committed:
Get Emails from Gmail then Clean and Dump to JSON

Put the client_secret file(find it in the API folder) under your own working path and run the .py file. It will raise a google account verification. Click continue, it will collect the email from your inbox and dump essentail information into JSON file

cliaker committed:
Create readme.md

cliaker committed:
Create readme.md

cliaker committed:
Realworld Email Data

Clean and in JSON style.

cliaker committed:
Update readme.md

cliaker committed:
Delete API and Data/email.json

Upload the wrong file

cliaker committed:
2200+ Realworld Email

SwordAndTea committed:
rename folders and positioned files to the corresponding folders

SwordAndTea committed:
delete unnecessary readme file
